docs/getting-started.md

```diff
@@ -0,0 +1,250 @@
+        .set('idCounter', state.get('idCounter') + 1);
+    case REMOVE_TODO:
+      return state
+        .update('todos',
+          todos => todos.filter(todo => todo.id !== action.payload )
+        );
+    default:
+      return state;
+  }
+}
+```
+
+If you're thinking "yuck, switch statements," remember that reducers are just functions — you can abstract away these details using helpers. Check out [redux-actions](https://github.com/acdlite/redux-actions) for an example of how to use higher-order functions to create reducers.
+
+## Creating a store
+
+Now we have some action creators and a reducer to handle them. The next step is to create our store.
+
+To create a store, pass a reducer to `createStore()`:
+
+```js
+import { createStore } from 'redux';
+import todoReducer from '../reducers/todos';
+const store = createStore(todoReducer);
+```
+
+Usually you'll have multiple reducers for different domains of data in your app. You can use the `combineReducers()` helper to combine multiple reducers into one:
+
+```js
+import { createStore, combineReducers } from 'redux';
+import * as reducers from '../reducers';
+const reducer = combineReducers(reducers);
+const store = createStore(reducer);
+```
+
+For example, if the object passed to `combineReducers()` looks like this:
+
+```js
+const reducers = {
+  todos: todoReducer,
+  counter: counterReducer
+};
+```
+
+It will create a reducer which produces a state object like this:
+
+```js
+const state = {
+  todos: todoState,
+  counter: counterState
+};
+```
+
+## Middleware
+
+Middleware is an optional feature of Redux that enables you to customize how dispatches are handled. Think of middleware as a certain type of plugin or extension for Redux.
+
+A common use for middleware is to enable asynchronous dispatches. For example, a promise middleware adds support for dispatching promises:
+
+```js
+dispatch(Promise.resolve({ type: ADD_TODO, payload: { text: 'Use middleware!' } }));
+```
+A promise middleware would detect that a promise was dispatched, intercept it, and instead dispatch with the resolved value at a future point in time.
+
+Middleware is very simple to create using function composition. We won't focus on how middleware works in this document but here's how you enable it when creating your store:
+
+```js
+import { createStore, applyMiddleware } from 'redux';
+// where promise, thunk, and observable are examples of middleware
+const store = applyMiddleware(promise, thunk, observable)(createStore)(reducer);
+```
+
+Yes, you read that correctly. [Read more about how middleware works here.](middleware.md)
+
+## Connecting to your views
+
+You'll rarely interact with the store object directly. Most often, you'll use some sort of binding to your preferred view library.
+
+Flux is most popular within the React community, but Redux works with any kind of view layer. The React bindings for Redux are available as react-redux — see that project for details on how to integrate with React.
```

Comment:
That's true :-) I was suggesting the switch because of the second part of the sentence.
In my mind the current wording suggests Flux is the "view layer", which seems wrong.

*Flux is most popular [as a view layer] within the React community, but Redux works with any kind of view layer.*

Anyways, great work on Redux and the new docs :+1: